terramate: 0.12.1 -> 0.13.0

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from https://github.com/terramate-io/terramate/releases.


meta.description for terramate is: Adds code generation, stacks, orchestration, change detection, data sharing and more to Terraform

meta.homepage for terramate is: https://github.com/terramate-io/terramate

meta.changelog for terramate is: https://github.com/terramate-io/terramate/releases/tag/v0.13.0


###### Updates performed
- Golang update

###### To inspect upstream changes

- [Release on GitHub](https://github.com/terramate-io/terramate/releases/tag/v0.13.0)

- [Compare changes on GitHub](https://github.com/terramate-io/terramate/compare/v0.12.1...v0.13.0)

###### Impact

<b>Checks done</b>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed
- found 0.13.0 with grep in /nix/store/7wn3ibi30yl17773a6sm61qayvi8b8qd-terramate-0.13.0
- found 0.13.0 in filename of file in /nix/store/7wn3ibi30yl17773a6sm61qayvi8b8qd-terramate-0.13.0

---

<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
2 total rebuild path(s)

2 package rebuild(s)

First fifty rebuilds by attrpath

terramate
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from the cache**:
```
nix-store -r /nix/store/7wn3ibi30yl17773a6sm61qayvi8b8qd-terramate-0.13.0 \
  --option binary-caches 'https://cache.nixos.org/ https://nixpkgs-update-cache.nix-community.org/' \
  --option trusted-public-keys '
  nixpkgs-update-cache.nix-community.org-1:U8d6wiQecHUPJFSqHN9GSSmNkmdiFW7GW7WNAnHW0SM=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The nixpkgs-update cache is only trusted for this store-path realization.)
For the cached download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A terramate https://github.com/r-ryantm/nixpkgs/archive/ad79ee1ac24485a3b5cd67f0b5f2e929b306468a.tar.gz
```
Or:
```
nix build github:r-ryantm/nixpkgs/ad79ee1ac24485a3b5cd67f0b5f2e929b306468a#terramate
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/7wn3ibi30yl17773a6sm61qayvi8b8qd-terramate-0.13.0
ls -la /nix/store/7wn3ibi30yl17773a6sm61qayvi8b8qd-terramate-0.13.0/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>terramate</li>
  </ul>
</details>

---

###### Maintainer pings

cc @dit7ya @asininemonkey for [testing](https://github.com/ryantm/nixpkgs-update/blob/main/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

> [!TIP]
> As a maintainer, if your package is located under `pkgs/by-name/*`, you can comment **`@NixOS/nixpkgs-merge-bot merge`** to automatically merge this update using the [`nixpkgs-merge-bot`](https://github.com/NixOS/nixpkgs-merge-bot).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

r-ryantm committed:
terramate: 0.12.1 -> 0.13.0

khaneliman commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 391159`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>terramate</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>terramate</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>terramate</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>terramate</li>
  </ul>
</details>

khaneliman committed:
terramate: fix darwin

http: TLS handshake error from 127.0.0.1:53166: remote error: tls: bad
certificate

khaneliman commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 391159`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>terramate</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>terramate</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>terramate</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>terramate</li>
  </ul>
</details>
